BRAD/agent.py

```
@@ -407,10 +411,17 @@ def invoke(self, query):
             self.state = self.reconfig()
             return True
             # continue
+        # Continue previous module
+        elif self.state['continue-module'] is not None:
+            route = self.state['continue-module'][0]
+            prinf("continue module is not None!")
```

Ruff

`prinf` may be undefined, or defined from star imports

(F405)